machine_management/control_plane_machine_management/cpmso-getting-started.adoc

```diff
@@ -6,37 +6,71 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-For AWS, {product-title} clusters that are created with version 4.12 or later use the Control Plane Machine Set Operator by default. 
+The process for getting started with the Control Plane Machine Set Operator depends on the state of the `ControlPlaneMachineSet` custom resource (CR) in your cluster. 
 
-To use this Operator on AWS clusters that are upgraded to {product-title} version 4.12 or later from an earlier version, you must xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[activate the Operator].
+Clusters with an active generated CR:: Clusters that have a generated CR with an active state use the Operator by default. No administrator action is required. 
+//+
+//AWS clusters that are created with {product-title} version 4.12 or later use the Control Plane Machine Set Operator by default. 
 
-For Azure and VMware vSphere clusters, you must xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-creating-cr_cpmso-getting-started[create a `ControlPlaneMachineSet` custom resource (CR)] and then xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[activate the Operator].
+Clusters with an inactive generated CR:: For clusters that include an inactive generated CR, you must review the CR configuration and xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[activate the Operator].
+//+
+//AWS clusters that are upgraded to {product-title} version 4.12 or later from an earlier version include an inactive generated CR.
+
+Clusters without a generated CR:: For clusters that do not include a generated CR, you must xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-creating-cr_cpmso-getting-started[create and activate a CR] with the appropriate configuration for your cluster.
+//+
+//For Azure and VMware vSphere clusters
+
+If you are uncertain about the state of the `ControlPlaneMachineSet` CR in your cluster, you can xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-checking-status_cpmso-getting-started[verify the Operator status]. 
```

Comment:
Linked to: https://github.com/openshift/openshift-docs/pull/52802/files#r1028380991